WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1454,6 +1471,10 @@ decisions = {
 			}
 			modifier = {
 				factor = 0
+				trait = tolerant_ruler
```

Comment:
Actually you are right, ingame terms we can't represent the subtleties of tolerating specific religions.